File: parte001/ex018_mayor_tres_numeros.js
// Escribir un algoritmo que acepte tres números enteros diferentes y muestre el mayor de ellos.

const numero1 = 5;
const numero2 = 10;
const numero3 = 15;

if (numero1 > numero2 && numero1 > numero3) {
    console.log("El número " + numero1 + " es el mayor.");
} else if (numero2 > numero1 && numero2 > numero3) {
    console.log("El número " + numero2 + " es el mayor.");
} else if (numero3 > numero1 && numero3 > numero2) {
    console.log("El número " + numero3 + " es el mayor.");
} else {
    console.log("Los tres números son iguales.");
}

File: parte001/ex018_mayor_tres_numeros.java
// Escribir un algoritmo que acepte tres números enteros diferentes y muestre el mayor de ellos.

import java.util.Scanner;

public class ex018_mayor_tres_numeros {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        int a, b, c;
        System.out.print("Digite el valor de A: ");
        a = entrada.nextInt();
        System.out.print("Digite el valor de B: ");
        b = entrada.nextInt();
        System.out.print("Digite el valor de C: ");
        c = entrada.nextInt();
        
        if (a > b && a > c) {
            System.out.println("A es el mayor");
        } else if (b > a && b > c) {
            System.out.println("B es el mayor");
        } else if (c > a && c > b) {
            System.out.println("C es el mayor");
        } else {
            System.out.println("Los números son iguales");
        }
        
        entrada.close();
    }
}
